website/docs/requests.mdx

```diff
@@ -121,20 +121,20 @@ You can lower the frequency your application downloads the `config JSON` by sett
 
 ### Use webhooks
 
-In a backend application, you might want to try using [Webhooks.](./advanced/notifications-webhooks.mdx) This way your application gets notified about changes and downloads the `config JSON` only when it is needed.
+In a backend application, you may want to consider using [Webhooks.](./advanced/notifications-webhooks.mdx) This way your application receives notifications about changes and downloads the `config JSON` only when needed.
 
 ### Call your backend instead of the ConfigCat CDN
 
 In the case of a frontend application, you can reduce the number of calls made towards the ConfigCat CDN by moving the evaluation logic from the frontend application to your backend.
 
 ### Use the ConfigCat Proxy to cache/proxy config JSON downloads
 
-The [ConfigCat Proxy](https://configcat.com/docs/advanced/proxy/proxy-overview/) allows you to host a feature flag evaluation and config JSON proxy/cache service within your own infrastructure.  
-The [CDN proxy](https://configcat.com/docs/advanced/proxy/endpoints/#cdn-proxy) feature allows you to proxy/cache config JSON downloads and the ConfigCat SDKs can be routed to the ConfigCat Proxy running in your own infrastructure greatly reducing the number of config JSON downloads.
+The [ConfigCat Proxy](/advanced/proxy/proxy-overview/) allows you to host a feature flag evaluation and config JSON proxy/cache service within your own infrastructure.  
+The [CDN proxy](/advanced/proxy/endpoints/#cdn-proxy) feature allows you to proxy/cache config JSON downloads, and the ConfigCat SDKs can be routed to the ConfigCat Proxy running in your own infrastructure, greatly reducing the number of config JSON downloads.
 
 ### Purchase additional quota
 
-You can purchase additional quota for your subscription to increase your limits. [Contact us](https://configcat.com/support/) for more information.
+You can purchase an additional quota for your subscription to increase your limits. [Contact us](https://configcat.com/support/) for more information.
```

Comment:
Same issue with the word "quota" as above.

Reply:
changed it